src/jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.tsx

```diff
@@ -77,6 +81,10 @@ export const DesktopArticleVideoPlayerContent: React.FC<DesktopArticleVideoPlaye
 		if (pipIcon) pipIcon.style.display = 'none';
 	}
 
+	if (preventRendering) {
+		return null;
+	}
```

Comment:
It's not recommended to have early returns before hooks as it may break rendering the component. See: https://legacy.reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
I would be more worried about returning null here (BE background is calling), but I assume we know that the null will be handled correctly.

Reply:
I get it, but there's a significant amount of code between the hooks and this condition.